test/unitTests.js

```diff
@@ -166,3 +166,40 @@ describe('Profile Test for Correct email, age, gender, instagram username, and p
 
 // Unit Tests for File Upload Feature
 
+
+//Unit Tests for Map Markers
+
+const city3 = "Toronto"
+const city4 = "Rome"
+const groupSize3 = 12
+const groupSize4 = 50
+const startDate3 = new Date("2020-11-30")
+const endDate3 = new Date("2020-12-02")
+
+describe('Test number of attractions returned: Toronto with group size 12',function(){
+    it('Recommended Attractions should return 4', async function(){
+        const response = await RecommendController.recommend(city3,groupSize3,startDate3,endDate3)
+        assert.equal(response.length,4);
+    })
+});
+
+describe('Test number of attractions returned: Toronto with group size 50',function(){
+    it('Recommended Attractions should return 4', async function(){
+        const response = await RecommendController.recommend(city3,groupSize4,startDate3,endDate3)
+        assert.equal(response.length,4);
+    })
+});
+
+describe('Test number of attractions returned: Rome with group size 12',function(){
+    it('Recommended Attractions should return 7', async function(){
+        const response = await RecommendController.recommend(city4,groupSize3,startDate3,endDate3)
+        assert.equal(response.length,7);
+    })
+});
+
+describe('Test number of attractions returned: Rome with group size 12',function(){
+    it('Recommended Attractions should return 2', async function(){
+        const response = await RecommendController.recommend(city4,groupSize4,startDate3,endDate3)
+        assert.equal(response.length,2);
+    })
+});
```

Comment:
It looks like here you are trying to see how many attractions would be returned with two different group sizes (I can see this in the response statements on lines 195 and 202). When you describe the unit test - both of them say with group size 12. Maybe change this just to make sure its clear what you are testing? I was confused why you did the same test twice before I read the rest of the code.